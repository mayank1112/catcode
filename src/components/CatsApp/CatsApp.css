.catsContainer {
    display: grid;
    grid-template-columns: auto auto;
  }

@media screen and (max-width: 768px) {
  .catsContainer {
    grid-template-columns: auto;
  }
}