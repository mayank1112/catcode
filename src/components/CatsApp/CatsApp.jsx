import CatGrid from "../CatGrid/CatGrid";
import CatList from "../CatList/CatList";
import './CatsApp.css';

const CatsApp = ({ cats, groupedCats, selectedCats, selectCat }) => (
    <div className='catsContainer' key={selectedCats}>
        <CatList
            cats={cats}
            groupedCats={groupedCats}
            selectedCats={selectedCats}
            selectCat={selectCat}
        />
        <CatGrid cats={cats} groupedCats={groupedCats} />
    </div>
);
export default CatsApp;
