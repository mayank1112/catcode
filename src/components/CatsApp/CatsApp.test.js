import Enzyme, { shallow } from 'enzyme';
import CatsApp from './CatsApp';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

Enzyme.configure({ adapter: new Adapter() });

it('<CatsApp> component should match snapshot', () => {
    const component = shallow(<CatsApp />);
    expect(toJson(component)).toMatchSnapshot();
});
