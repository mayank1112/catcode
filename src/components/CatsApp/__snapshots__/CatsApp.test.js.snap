// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CatsApp> component should match snapshot 1`] = `
<div
  className="catsContainer"
>
  <CatList />
  <CatGrid />
</div>
`;
